k8sclient.go

```diff
@@ -72,6 +72,13 @@ func getk8sJob(job Job) *batchv1.Job {
 			TTLSecondsAfterFinished: &tTLSecondsAfterFinished,
 		},
 	}
+
+	// Merge securityContext and resources from podconfig only if it's defined
```

Comment:
Let's move this to the block below (same as Affinity and ImagePullSecrets and assign values from "copy" and not "podconfig"

Reply:
changed tested and pushed